framework/test/fixtures/accounts.js

```diff
@@ -62,16 +60,16 @@ accounts.mem_accountsFields = [
 	'asset',
 	'membersPublicKeys',
 	'votedDelegatesPublicKeys',
+	'secondPublicKey',
```

Comment:
why not remove this one?

Reply:
as per Pablo's suggestion, the migration to remove these columns was not addressed in this PR, we will remove these columns from the database in the other issue which address the migration of second signature to multi-sign. I have already update that issue with the migration information